packages/hash/frontend/src/blocks/page/Comments/placeholderPlugin.tsx

```diff
@@ -0,0 +1,50 @@
+import Typography from "@mui/material/Typography";
+import { Schema } from "prosemirror-model";
+import { Plugin } from "prosemirror-state";
+import { Decoration, DecorationSet } from "prosemirror-view";
+import { RenderPortal } from "../usePortals";
+
+export const placeholderPlugin = (renderPortal: RenderPortal, text: string) =>
```

Comment:
Maybe we could put a comment line explaining why we have another `placeholderPlugin`,  instead of using `src/blocks/page/createPlaceholderPlugin` (e.g. "We need a tiny version of that plugin just to show the placeholder without caring about state" or sth like that)

Reply:
Added a small comment explaining it's just a simplified version of the createPlaceholderPlugin in 6633fc7f59956c93bf04ec909e467d70e8f4e1ff (and also renamed the plugin to commentPlaceholderPlugin).